modules/ROOT/pages/tutorial-setup.adoc

```diff
@@ -1,70 +1,136 @@
 = Prerequisites
 
+== Virtualization with `libvirt`
+
 Those tutorials are written targeting a Linux environment with a working `libvirt` setup and hardware virtualization support via `KVM`. For instructions to setup `libvirt` and `KVM` you may refer to the link:https://docs.fedoraproject.org/en-US/quick-docs/getting-started-with-virtualization/[Getting started with virtualization] guide from Fedora. Although this setup guide is focused on Fedora, those tutorials should work on any distribution with `libvirt` installed.
 
-== Downloading a Fedora CoreOS QCOW2 image
+== CoreOS tools
+
+For those tutorials, we will need the following tools:
+
+  * `coreos-installer`: Tto download the latest Fedora CoreOS QCOW2 image,
+  * `fcct`: To generate Ignition configuration from Fedora CoreOS Config files,
+  * `ignition-validate`: To validate Ignition configuration files.
 
-First of all, we have to download the latest stable release of Fedora CoreOS. As we will use `libvirt`, we will download an archive containing a QEMU image from link:https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[getfedora.org]:
+=== Setup with `podman` or `docker`
+
+All of the tools required to work with Fedora CoreOS are available from containers hosted on link:https://quay.io/[quay.io]:
 
 [source,bash]
 ----
-RELEASE="32.20200715.3.0" # Update this with the latest release from https://getfedora.org/en/coreos
-curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
-curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig
+podman pull quay.io/coreos/coreos-installer:release
+podman pull quay.io/coreos/fcct:release
+podman pull quay.io/coreos/ignition-validate:v2.4.1
 ----
 
-Once the archive has been downloaded, make sure to verify its integrity by following the instructions available by clicking on the `Verify signature & SHA256` button. You will have to download the checksum file, the signature and Fedora GPG keys to verify your download:
+You can then use `coreos-installer` to download the latest stable image with:
 
 [source,bash]
 ----
-curl https://getfedora.org/static/fedora.gpg | gpg --import
-gpg --verify fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig
+podman run --pull=always --rm -ti -v .:/data:Z \
+    quay.io/coreos/coreos-installer:release download \
+    -p qemu -f qcow2.xz --decompress -C /data/
 ----
 
-Once you have verified the archive, you can extract it with:
+To make the tutorial simpler, you should rename the image that we have just downloaded to a shorter name:
 
 [source,bash]
 ----
-unxz fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+mv fedora-coreos-*-qemu.x86_64.qcow2 fedora-coreos.qcow2
```

Comment:
This can be a bit dangerous singe it would match all files of that pattern. It would make me feel better if we instruct the user to make a new directory to operate in first.